helm-chart/binderhub/schema.yaml

```diff
@@ -255,17 +255,6 @@ properties:
       manifest as either the binder pod going into `Error` or `CrashLoopBackoff` states, or in
       some special cases, the binder pod running but... just doing very random things. Be careful!
 
-  cors: &cors-spec
-    type: object
-    additionalProperties: false
-    description: |
-      TODO
-    properties:
-      allowOrigin:
-        type: [string, "null"]
-        description: |
-          TODO
-
```

Comment:
This is associated with line 28 in the schema, writing that the `cors` field is required. Remove it as well and anyone with this config will get an error, making anyone doing a migration get a hard error.

Another option, is to make it so that you let the schema allow this, and let NOTES.txt do a `print "migration error message" | fail` which can be more helpful.